source/guides/contributing/layout.md

```diff
@@ -0,0 +1,15 @@
+Denizen's Project Layout
+------------------------
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+**DenizenScript** is a language run by **Denizen**. The most popular implementation of Denizen is the Spigot implementation, which is the jar you've most likely downloaded many times. As of the time of writing, it is the *only* maintained impl.
+
+Any Denizen implementation depends on **DenizenCore**, which is the main set of tools that powers every language feature. It does include some content that isn't specific to Minecraft implementations, such as basic ElementTag manipulation and data script containers.
```

Comment:
Backwards implication - it's not that the core happens to contain some things, it's that DenizenCore does (or should) contain any/all features that aren't implementation-specific.